src/validation.h

```diff
@@ -138,7 +139,7 @@ static const int DEFAULT_STOPATHEIGHT = 0;
 
 struct BlockHasher
 {
-    size_t operator()(const uint256& hash) const { return hash.GetCheapHash(); }
+    size_t operator()(const uint256& hash) const { return ReadLE64(hash.begin()); }
```

Comment:
nit: Could benefit from a comment referencing `GetCheapHash` to explain the implementation.

Reply:
Okay. Got 3 utACKs so will fix this if another rebase becomes necessary or other changes are suggested.

Reply:
Could add a commit with "doc: bla..." on top of the branch (not touching the existing commits)?

Reply:
Good idea. Done!